app-doc/gimp-help/gimp-help-2.10.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
```

Comment:
Is it really not python3 compatible? I suspect 2.7 might've been used due to libxml2 dep. Could you test dropping python2 support, and with 3 series?

Reply:
That python2-only support will become a problem for us real soon. You should look into patching it somehow.